### PR TITLE
adds readme with wireframes

### PR DESCRIPTION
**Pull request by hp1046**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,43 @@
...
+
+A portfolio site showcasing my work.
+
+## Wireframes
+
+Wireframes for the planned layout of my project.
+
+### Home Page
+
+![Wireframe of Home Page](wireframes\home.jpg)
```
Comment (axamei) [inline README.md]: You want to use `/` instead of `\`

File: README.md [hunk 2]
```diff
@@ -1,2 +1,43 @@
...
+
+Navigation Menu - Will include links to the home, education and certification, and work experience and professional development pages
+
+Main Content - Will have about me, my philosophy, and contact me sections
+
+Footer - Will include copyright information
+
+### Education and Certification Page
+
+![Wireframe of Education and Certification](wireframes\education-and-certification.jpg)
```
Comment (axamei) [inline README.md]: You want to use `/` instead of `\`

File: README.md [hunk 3]
```diff
@@ -1,2 +1,43 @@
...
+
+Navigation Menu - Will include links to the home, education and certification, and work experience and professional development pages
+
+Main Content - Will have information about my educational background and state certification and endorsements
+
+Footer - Will include copyright information
+
+### Work Experience and Professional Development Page
+
+![Wireframe of Home Page](wireframes\work-experience-and-professional-development.jpg)
```
Comment (axamei) [inline README.md]: You want to use `/` instead of `\`

**Review by axamei — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (hp1046) [reply README.md]: @axamei Fixed! Thank you!

File: README.md (hunk 2 above)
Comment (hp1046) [reply README.md]: @axamei Fixed! Thank you!

File: README.md (hunk 3 above)
Comment (hp1046) [reply README.md]: @axamei Fixed! Thank you!

**Review by axamei — APPROVED** [review]

**Pull request merged**